GirafRest/Controllers/UserController.cs

```diff
@@ -448,11 +445,6 @@ public async Task<ActionResult> GetCitizens(string id)
 
             //sort function for users in citizens since the list needs to be sorted by name... issue#697
             citizens.Sort();
-           
```

Comment:
Kan ikke lige gennemskue hvorfor denne og den tilsvarende med guardians i linje 491 er unnecessary

Reply:
Det kan jeg umiddelbart heller ikke. Det eneste jeg kunne se som værende en mulighed skulle være fordi det reelt set ikke er noget der skal afgive en fejl. Er den der måske kun for at undgå unexpected behavior?

Reply:
kan se de er der tidligere

Reply:
Issue #826 (https://github.com/aau-giraf/weekplanner/issues/826) beskriver at det er uønsket, at der kommer en fejlbesked i terminalen hvis en bruger ikke har nogle Citizens. Det er (i følge issue'en) uproblematisk ikke at have nogle citizens. Er det eventuelt en exception der stadig skal kastes i GetCitizens() og GetGuardians(), men gribes et andet sted, eksempelvis i api_client eller i weekplanner?